src/db/collections/outputs/mod.rs

```diff
@@ -514,7 +641,12 @@ mod analytics {
                                 } },
                             ],
                             "nft_changed": [
-                                { "$match": { "output.nft_id": { "$ne": NftId::implicit() } } },
+                                { "$match": { 
+                                    "$and": [
+                                        { "output.nft_id": { "$exists": true } },
```

Comment:
This means that our previous computation was wrong, right? I've been seeing fishy numbers in the Grafana dashboard :grimacing:.